lib/Data/Binary.hm

```diff
@@ -53,22 +53,39 @@ toList = ffi1 :erlang :binary_to_list
 toTerm :: Binary -> Term
 toTerm = ffi1 :erlang :binary_to_term
 
+-----------------------------------------------------------------------------
+-- | Operators
+
+-- | Get Byte from Binary
+at :: Integer -> Binary -> Integer -- TODO: exception?
+at pos bin = ffi2 :binary :at bin pos
```

Comment:
The args are unnecessary for FFI functions.  Pls change to `at = ffi2 :binary :at`.